config/schema.json

```diff
@@ -38,6 +38,12 @@
     "packageName": {
       "$ref": "#/definitions/packageName"
     },
+    "packageConfig": {
+      "$ref": "#/definitions/packageConfig"
+    },
+    "customElementsManifestConfig": {
+      "$ref": "#/definitions/customElementsManifestConfig"
+    },
```

Comment:
I wonder if it might be better for us to include this under the "build" configuration section instead since it relates to the build? I'm ok with it as-is if we want to expose a new top-level command that executes just the manifest generation so it could be used on its own. That could be something else to consider with this change. Thoughts?

Reply:
I could probably implement a top-level command.  I think having it as part of the build process is the most streamlined approach, but if a project did want to provide their own config file to customize generation, it might also make sense for that project to run the command separately.  Should the `disabled` flag be moved to a property of the build configuration, since that would just prevent it from running as part of the build?  Or keep it here and just rename it to something like `disableBuildGeneration` or change it to a string that would be either `manual` or `auto`.

Reply:
I was thinking we would keep this as part of the build command process, but also expose the functionality as a separate command in case we need to run it on its own (which may be useful for other tools that need to use only the manifest output without running a full build).

Reply:
Added a new command, which by default would give you the same manifest as the build, output to the current directory.  I kept support for the custom config file, though I feel like if a project's needs were unique enough to require one they might be better off just consuming the analyzer directly.